lib/src/widgets/ds_bottomsheet_countries.widget.dart

```diff
@@ -0,0 +1,400 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:get/get.dart';
+import '../../blip_ds.dart';
+import '../models/ds_country.model.dart';
+import 'fields/ds_app_search_input.widget.dart';
+
+abstract class DSBottomSheetCountries {
+  static final showClearButton = RxBool(false);
+  static final _filterCountries = RxList<DSCountry>([]);
+  static final TextEditingController controller = TextEditingController();
+  static final selectedCountry = Rxn<DSCountry>();
+
+  static show() async {
+    _filterCountries.assignAll(listCountries);
+    return _bottomSheetCountries();
+  }
+
+  static _bottomSheetCountries() {
+    return DSBottomSheetService(
+      fixedHeader: Column(
+        children: [
+          Padding(
+            padding: const EdgeInsets.only(left: 16.0),
+            child: SafeArea(
+              top: false,
+              bottom: false,
+              child: Row(
+                mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                children: [
+                  DSHeadlineLargeText(
+                    'País',
+                  ),
+                  DSIconButton(
+                    onPressed: () {
+                      Get.back();
+                    },
+                    icon: const Icon(DSIcons.close_outline,
+                        color: DSColors.neutralDarkRooftop),
+                  ),
+                ],
+              ),
+            ),
+          ),
+          const DSDivider(),
+          Padding(
+            padding: const EdgeInsets.fromLTRB(
```

Comment:
You could use ```EdgeInsets.symmetric``` since vertical (top-bottom) and horizontal (left - right) values are equal.

Reply:
Done.